dbt/adapters/impala/cloudera_tracking.py

```diff
@@ -65,6 +88,7 @@ def _tracking_func(data):
             logger.debug(f"Error reading tracking config. {err}")
             logger.debug("Disabling usage tracking due to error.")
             usage_tracking = False
+            return
```

Comment:
If exception we return to dbt main thread and continue with the rest of execution without worrying about tracking anymore.

Reply:
ok.